profile2/impl/src/java/org/sakaiproject/profile2/logic/ProfileConnectionsLogicImpl.java

```diff
@@ -510,7 +518,7 @@ public int getOnlineStatus(String userUuid) {
 			throw new SecurityException("You must be logged in to get a connection list.");
 		}
 		
-		List<User> users = new ArrayList<User>();
+		List<User> users = new ArrayList();
```

Comment:
Is that how it works? I thought it took the left hand side as the type - I thought that was the point. That's a bit crap.